Symfony/CS/Tests/Fixer/PSR2/VisibilityFixerTest.php

```diff
@@ -405,4 +405,27 @@ class Foo
 
         $this->makeTest($expected, $input);
     }
+
+    public function testFixesVarDeclarationWhereValueIsArrayWithMoreThanOneValue()
+    {
+        $expected = <<<'EOF'
+<?php
+class Foo
+{
+  public $foo = array('foo');
+  public $bar = array('bar', 'baz');
+}
+EOF;
+
+        $input = <<<'EOF'
+<?php
+class Foo
+{
+  var $foo = array('foo');
+  var $bar = array('bar', 'baz');
```

Comment:
@keradus 

```php
$prevIndex = $tokens->getPrevTokenOfKind($index, array(';', ','));
$nextIndex = $tokens->getNextTokenOfKind($index, array(';', ','));

// Fails because this condition evaluates to false
if (
    (!$prevIndex || !$tokens[$prevIndex]->equals(',')) &&
    (!$nextIndex || !$tokens[$nextIndex]->equals(','))
) {
    $this->applyAttribs($tokens, $index, $this->grabAttribsBeforePropertyToken($tokens, $index));
}
```